### ISSUE TITLE
v.1.0.6 unable to verify devices.

### ISSUE THREAD

**Issue by AtomicRPM** [vector-im/element-android#2113]
**Describe the bug**
Pulled latest release of Element-android v1.0.6 and built app using Android Studio. Installed the built apk on two android 10 phones.  User 1 and user 2 sign into homeserver.  User 1 tries to verify user 2 device.  User 2 receives invite to private room instead of device verification request.

Homeserver is v1.19.1

**Comment by bmarty** [comment]
Yes, this is the expected behavior by design

**Comment by AtomicRPM** [comment]
OK.  I guess I am missing something then.  How does User 1 verify User 2's device?  The "expected behavior by design" is not intuitive nor does it inform User1 that for a device verification to succeed a private room must be created which results in a verification request never being received by User 2.  This creates a lot of confusion and is far from how device a verification occurred in Riot, and honestly is not the expected behavior.

So what is supposed to happen in the private room?  A second device verification should be sent?

Could you please list the steps required to verify two devices.

**Comment by witchent** [comment]
No, in the created room the verification request should be there already, so the second person just has to join the room and click on accept.
That is except if you have "never send encrypted messages to unverified devices" turned on.

I am not sure if it was always like this, but I think this behavior is really bad UX as well.
You can get bitten by https://github.com/vector-im/element-web/issues/13656 and https://github.com/vector-im/element-web/issues/13600, both which just happened to me in Element Android, which makes you end up with two chats with the same person (bad enough) but also unable to verify except for if you figure out the cause. 
As mentioned by @AtomicRPM, this really creates confusion for everyone involved.
